yarn.lock

```diff
@@ -2471,6 +2487,24 @@
     eventemitter3 "^4.0.0"
     rxjs "^6.5.4"
 
+"@polkadot/api@1.8.0-beta.4", "@polkadot/api@^1.8.0-beta.4":
```

Comment:
Isn't that a problem to have `1.8.0-beta.4` and `^1.8.0-beta.4`?
I've had this in the past and couldn't figure out whether it was ok or not..

Reply:
No. The most important thing to have is one single `@polkadot/api` in yarn.lock

Reply:
Which is actually not the case now